### ISSUE TITLE
Please provide the GPU configs and training time. 3Q

### ISSUE THREAD

**Issue by Currycurrycurry** [bcmi/SimTrans-Weak-Shot-Classification#1]

**Comment by Currycurrycurry** [comment]
@chenbys @ustcnewly May you also provide the specific params to get the best accuracy? Appreciate a lot :)

**Comment by chenbys** [comment]
The learning rates are about [5e-5,5e-4,5e-3] and batch sizes are about [16,32,64,128]. The overall performances are not hard to achieve. Subtle tuning may need for various environments.

**Comment by Currycurrycurry** [comment]
> The learning rates are about [5e-5,5e-4,5e-3] and batch sizes are about [16,32,64,128]. The overall performances are not hard to achieve. Subtle tuning may need for various environments.
Thanks a lot 👍
